src/createButton.ts

```diff
@@ -12,11 +14,26 @@ export interface ButtonFactory extends ComposeFactory<Button, ButtonOptions> { }
 
 const createButton: ButtonFactory = createWidget
 	.mixin({
-		mixin: createFormFieldMixin
+		mixin: createFormFieldMixin,
+		aspectAdvice: {
+			before: {
+				getNodeAttributes(this: Button, overrides: VNodeProperties = {}) {
+					if (this.state.classes) {
```

Comment:
Don't you want this to run, even if `this.state.classes` isn't true (e.g. always have `classes[css.button] = true;`)?  The code coverage analysis shows that the block below isn't being covered.